src/%IPM/Utils/Module.cls

```diff
@@ -1118,15 +1118,15 @@ ClassMethod LoadNewModule(pDirectory As %String, ByRef pParams, pRepository As %
 		Set tSC = $system.OBJ.Load(pDirectory_"module.xml",$Select(tVerbose:"d",1:"-d"),,.tLoadedList)
 		$$$ThrowOnError(tSC)
 		
-		Set tFirstLoaded = $$$lcase($Order(tLoadedList("")))
+		Set tFirstLoaded = $Order(tLoadedList(""))
```

Comment:
why it was changed here?

Reply:
This is to make sure that packages are not recognized all in lower case. They should be kept in their original text when installed by IPM

Reply:
zpm packages have to be lowercase only, there are no reasons to change it, there were a few reasons, why we come to this decision

Reply:
Why zpm packages needs to be lowercase only?
We have a good reason to change it since package name themselves should be kept in their original form.

Reply:
naming convention
every where it should be normalized to one form, for search, for store any typo, should still be one case
dependency and so on
any reasons to break it?

Reply:
The naming convention should be kept to its original name, without casting to lower case. If any package's name breaks things by casting to lower case, that's a bug in other places that we need to fix, not to fix the name itself.

Reply:
Who is decided so?

Reply:
This project was driven by Community, and rules was set by community, by reasons,
And InterSystems, just don't care about it